### PR TITLE
fix: display dropdown of flags on mobile

### PR DESCRIPTION
**Pull request by GusBaamonde**
dropdown-flags-mobile

Related to: https://github.com/FromDoppler/Doppler-UI-System/pull/314

### CODE REVIEW

File: src/components/form-helpers/form-helpers.css [hunk 1]
```diff
@@ -4,10 +4,16 @@
   width: 100%;
 }
 
+.intl-tel-input {
+  position: relative;
+  display: inline-block;
+}
+
```
Comment (andresmoschini) [inline src/components/form-helpers/form-helpers.css:11]: @juanasas do you think that we also need it in Forms Generator?

File: public/index.html [hunk 2]
```diff
@@ -14,7 +14,7 @@
     <link
       rel="stylesheet"
       type="text/css"
-      href="https://cdn.fromdoppler.com/doppler-ui-library/v3.51.3/css/styles.css"
+      href="https://cdn.fromdoppler.com/doppler-ui-library/build887/css/styles.css"
```
Comment (cbernat) [inline public/index.html:17]: Please check not to leave the build here.

**Comment by cbernat** [comment]
Test this please.

**Comment by cbernat** [comment]
To test this link https://cdn.fromdoppler.com/doppler-webapp/build2396/#/signup

**Comment by cbernat** [comment]
> To test this link https://cdn.fromdoppler.com/doppler-webapp/build2396/#/signup

Works great on my phone! (old Android)

**Pull request merged**